schemas/json/theme.json

```diff
@@ -533,48 +546,57 @@
 				{ "$ref": "#/definitions/settingsPropertiesCustom" }
 			]
 		},
-		"settingsPropertiesComplete": {
+		"settingsPropertiesNames": {
+			"enum": [
+				"appearanceTools",
+				"border",
+				"color",
+				"layout",
+				"spacing",
+				"typography",
+				"custom"
+			]
+		},
+		"settingsPropertiesNested": {
 			"type": "object",
 			"allOf": [
 				{
 					"$ref": "#/definitions/settingsProperties"
 				},
 				{
-					"properties": {
-						"appearanceTools": {},
-						"border": {},
-						"color": {},
-						"layout": {},
-						"spacing": {},
-						"typography": {},
-						"custom": {}
-					},
-					"additionalProperties": false
+					"$ref": "#/definitions/settingsBlocksPropertiesComplete"
 				}
-			]
+			],
+			"propertyNames": {
+				"anyOf": [
+					{ "$ref": "#/definitions/settingsPropertiesNames" },
+					{ "$ref": "#/definitions/settingsBlocksNames" }
+				]
+			}
 		},
 		"settingsBlocksPropertiesComplete": {
 			"type": "object",
 			"properties": {
 				"core/archives": {
-					"description": "Archive block. Display a monthly archive of your posts. This block has no block-level settings",
-					"additionalProperties": false
```

Comment:
The archives block does not have any block-level settings, so we should probably update this to add `"type": "object"` rather than removing `"additionalProperties": false`

Reply:
Right, that makes sense to me!